etl/steps/data/garden/un/2024-03-14/un_wpp_most.meta.yml

```diff
@@ -0,0 +1,15 @@
+tables:
+  population:
+    variables:
+      age:
+        title: Age-group with the highest population
+        unit: ""
+        description_short: |-
+          Age of the five-year population age-group with the highest population
```

Comment:
I find it a bit redundant to use "age" twice in the sentence.

Maybe something like

> Five-year age group with the highest population.

maybe that's too simple?

Another example:

> The largest five-year population age group.

Reply:
Good point - yours is much better!